web/annotation_layer_builder.css

```diff
@@ -342,7 +342,7 @@
     display: inline-block;
   }
 
-  .popup > .header h1 {
```

Comment:
You mentioned that the `h1` caused a warning in Firefox, but which warning exactly and where is it visible because I haven't seen it before?

Changing this to a `span` element makes the title no longer bold as indicated by the failing reference tests. Is that expected, or do we need to add some styling to make it bold here so it matches the current situation?

Reply:
The warning is defined here:
https://searchfox.org/mozilla-central/source/dom/locales/en-US/chrome/dom/dom.properties#487
and the MDN reference about it:
https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/Heading_Elements#specifying_a_uniform_font_size_for_h1